### ISSUE TITLE
F99 - What does "STR key" mean?

### ISSUE THREAD

**Issue by 8845musign** [w3c/wcag#1620]
Hi

In [F99](https://www.w3.org/WAI/WCAG21/Techniques/failures/F99), there is a section labeled "STR key".

> Viewing page scripts and searching for typical keyboard event handlers like document.addEventListener('keydown' ...) or the presence of the .keycode attribute may establish the presence of scripts that intercept keyboard shortcuts without modification keys like ALT or STR being held down at the same time.

I can't find it on the 109 and 104 keyboard layout I'm familiar with.

In context, did they miswrite the Ctrl and STR keys?

Many thanks
hiroki

**Comment by alastc** [comment]
@detlevhfischer, I seem to remember you were behind this one? Is the STR key a German thing?

**Comment by JAWS-test** [comment]
Strg (and not STR) is the german translation of Ctrl key

**Comment by 8845musign** [comment]
@alastc @JAWS-test 

I understand.
Thank you for answering my question!

**Comment by detlevhfischer** [comment]
Oops - I created PR #1631 to change STR tp Ctrl

**Comment by alastc** [comment]
Thanks Detlev, the change shouldl show up with the next publish of the materials.
